tests/basic/storage_pool_test.go

```diff
@@ -384,6 +384,7 @@ func roundUpValue(toRound uint64) uint64 {
 
 func poolResizeIsInProgress(poolToBeResized *api.StoragePool) bool {
 	poolSizeHasBeenChanged := false
+	waitCount := 20
```

Comment:
Why to wait 20 minutes for  pool resize operation to be succeed ? It seems this value is high for resize operation..

Reply:
reduced to 5 mins